homeassistant/components/zwave_js/services.yaml

```diff
@@ -232,7 +232,7 @@ ping:
 
 reset_meter:
   name: Reset meter(s) on a node
-  description: Resets the meter(s) on a node.
+  description: The service will reset all meters on the same endpoint of the entity(s) you target, so you should only choose one meter entity for a given node, endpoint, and meter type (if applicable). If you target multiple entities on the same node, endpoint, and meter type, you will trigger multiple reset calls which will generate unnecessary traffic on your network.
```

Comment:
Is it necessary to explain this? It doesn't really matter that you one-time have some extra traffic?

Reply:
I can't speak to how frequently people reset their meters, but I take your point. We can withdraw this - do you think it's worth fixing via the Debouncer like I suggested or do you think it's not worth the trouble?

Reply:
Also do we want to keep it in the docs? Or should we not mention it at all?

Reply:
I think that it's not worth mentioning.